javascript/javascript3/week1/computers.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
Nice, I like that you have a good mix of nested properties and arrays 😄

Reply:
thanks